ts/backend/src/app/features/controller/submission.controller.mts

```diff
@@ -187,51 +209,36 @@ export class SubmissionController extends Controller {
     }
 
     const benchmarkId = req.query['benchmark']
-    const submissionIds = req.query['uuid']?.split(',')
     const submittedBy = req.query['submitted_by']
 
     const sql = SqlService.getInstance()
 
-    let whereScores = sql.fragment`scores IS NOT NULL`
+    // per default, list all public submissions
     let wherePublic = sql.fragment`public = true`
```

Comment:
bug:
```suggestion
    let wherePublic = sql.fragment`published = true`
```

Reply:
We should add some test coverage, at least via https://github.com/flatland-association/flatland-benchmarks/issues/255